roles/backend_setup/tasks/vdo_create.yml

```diff
@@ -1,4 +1,16 @@
 ---
+- name: Set pyyaml package name
+  set_fact:
+    pyyaml_name: python-yaml
+- name: Overwrite pyyaml package name for non-Debian
+  set_fact:
+    pyyaml_name: PyYAML
```

Comment:
On RHEL and Fedora the package name is python2-pyyaml. So this will fail, you can actually remove the set_fact. And also the overwriting part. And use the package name directly and check for OS. For example:

package: 
    name: python-yaml
    state: present
when: ansible_distribution == "Debian" or ansible_distribution == "Ubuntu"